sharktank/tests/layers/rotary_embedding_v2_test.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2025 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import torch
+
+from sharktank.layers.rotary_embedding_v2 import RotaryEmbeddingV2Layer
+from transformers.models.llama.modeling_llama import (
+    LlamaRotaryEmbedding,
+    apply_rotary_pos_emb,
+)
+from transformers import LlamaConfig
+import pytest
+
+torch.manual_seed(123456)
```

Comment:
Like that tests don't get a frozen seed. If the execution order of tests changes their seed will change.
```
@pytest.fixture
def seed_torch_rng():
    with torch.random.fork_rng():
        yield

def test_thing(seed_torch_rng):
    ...
```